src/android/com/adobe/phonegap/push/PushPlugin.java

```diff
@@ -171,19 +176,25 @@ private void createDefaultNotificationChannelIfNeeded(JSONObject options) {
   }
 
   @Override
-  public boolean execute(final String action, final JSONArray data, final CallbackContext callbackContext) {
+  public boolean execute (
```

Comment:
Codacy found an issue: [The method 'execute(String, JSONArray, CallbackContext)' has an NPath complexity of 430, current threshold is 200](https://app.codacy.com/gh/havesource/cordova-plugin-push/pullRequest?prid=6101348)

Reply:
Will investigate later.